javascript-exercises-2/exercise-10/style.css

```diff
@@ -0,0 +1,6 @@
+#inputForm {
```

Comment:
Should use class to style instead of ID https://google.github.io/styleguide/htmlcssguide.html#ID_Selectors